app_build_suite/build_steps/helm.py

```diff
@@ -338,12 +355,11 @@ class HelmChartMetadataGenerator(BuildStep):
     _key_namespace_singleton = "namespaceSingleton"
     _key_gpu_instances = "gpuInstances"
     _key_fixed_namespace = "fixedNamespace"
-    _key_chart_file = "chartFile"
-    _key_digest = "digest"
-    _key_date_created = "dateCreated"
-    _key_chart_api_version = "chartApiVersion"
-    _key_api_version = "apiVersion"
     _key_annotations = "annotations"
+    _key_annotation_metadata_url = "application.giantswarm.io/metadata"
+    _key_annotation_readme_url = "application.giantswarm.io/readme"
+    _key_annotation_schema_url = "application.giantswarm.io/values-schema"
+    _value_annotation_catalog_base_url = "https://giantswarm.github.io/"
```

Comment:
I think we should avoid any "giantswarm-ism" here. Let's think about other potential users of this tool that are outside of giantswarm. I'm OK with this being a config option in the config file - let's just test at config loading if the value is there (if at all metadata build is requested).